### PR TITLE
docs(ava): optimize and improve ava readme-en

### PR DESCRIPTION
**Pull request by chenying2929**
1. modify the npm link error in the en documentation.
2. add intro section.
3. optimize cases.
4. synchronize the contribution section with the global document.

### CODE REVIEW

File: packages/ava/README.md [hunk 1]
```diff
@@ -8,11 +8,22 @@
...
+## Introduction
+
+[@antv/ava](https://www.npmjs.com/package/@antv/ava) is the core JS package of AVA, which contains four main modules:
+
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">data (Data Processing)</span>: Data Processing Module. Used for statistical analysis and processing of datasets.
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">insight (Auto Insight)</span>: Automatic Insights Module. Automatically discover data insights from multi-dimensional data.
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">ckb (Chart Knowledge Base)</span>: Chart Knowledge Base Module. Based on empirically derived knowledge and observations about the various fundamentals of visualization and charts, it is the cornerstone of intelligent chart recommendations.
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">advisor (Chart Recommendation)</span>: Chart Recommendation Module. Recommend chart types and specific chart detail settings based on data and analysis needs, as well as chart optimization for existing charts.
```
Comment (pddpd) [inline packages/ava/README.md:22]: Should we reorder these modules in the order of use or the first letter?

File: packages/ava/README.md [hunk 2]
```diff
@@ -22,35 +33,40 @@ $ npm install @antv/ava --save
...
-// Chart Knowledge Base
-const myCkb = ckb();
 
-// Data Processing and Analysis
+// 1. Data Processing Module
 const df = new DataFrame(data);
 const dataInfo = df.info();
 
-// Data Insight
+// 2. Automatic Insights Module
```
Comment (pddpd) [inline packages/ava/README.md:54]: L41 capitalizes the first letter of the first word only, L50 and L54 capitalize the first letter of all words, plz unify them.
https://github.com/antvis/AVA/pull/704/files#diff-884bd56149db70e7927ab4546a4ef2af42fa374ef9818a5e6f17414b4c8bacf2R41

**Review by pddpd — CHANGES_REQUESTED** [review]
The PR contains 4 commits, plz improve the commit number.

File: packages/ava/README.md (hunk 1 above)
Comment (chenying2929) [reply packages/ava/README.md:22]: I introduced the basic process of data processing and data analysis from shallow to deep. Global's readme and ava's Chinese version are currently following this approach.

**Comment by chenying2929** [comment]
> The PR contains 4 commits, plz improve the commit number.
done

File: packages/ava/README.md (hunk 1 above)
Comment (pddpd) [reply packages/ava/README.md:22]: discussed in person, we will continue to improve in the future PR

**Review by pddpd — APPROVED** [review]
LGTM

**Pull request merged**